portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase

```diff
@@ -180,6 +180,100 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-125802. The user can change the order of the page and child without affect the indentation in Navigation Menu."
+	@priority = "4"
+	test ChangeOrderOfPagesInNagivationMenu {
+		task ("Add three Pages via JSON") {
+			for (var n : list "1,2,3") {
+				JSONLayout.addPublicLayout(
+					groupName = "Test Site Name",
+					layoutName = "Test Page Name ${n}",
+					type = "content");
+
+				JSONLayout.publishLayout(
+					groupName = "Test Site Name",
+					layoutName = "Test Page Name ${n}");
+			}
+		}
+
+		task ("Add three children in the pages via JSON") {
+			for (var n : list "1,2,3") {
+				JSONLayout.addPublicLayout(
+					groupName = "Test Site Name",
+					layoutName = "Child Page ${n}",
+					parentLayoutName = "Test Page Name ${n}");
+			}
+		}
+
+		task ("Add a Nagivation Menu") {
+			JSONSitenavigation.addSiteNavigationMenu(
+				groupName = "Test Site Name",
+				siteNavigationMenuName = "Navigation Menu Name");
+		}
+
+		task ("Navigate to the navigation menu") {
+			NavigationMenusAdmin.openNavigationMenusAdmin(siteURLKey = "test-site-name");
+
+			LexiconEntry.gotoEntry(rowEntry = "Navigation Menu Name");
+		}
+
+		task ("Add the parents and children pages to the navigation menu") {
+			NavigationMenusAdmin.addItem(
+				item = "Page",
+				pageNames = "Test Page Name 1,Child Page 1,Test Page Name 2,Child Page 2,Test Page Name 3,Child Page 3");
+
+			for (var pageName : list "Test Page Name 1,Child Page 1,Test Page Name 2,Child Page 2,Test Page Name 3,Child Page 3") {
+				NavigationMenusAdmin.viewItem(itemName = "${pageName}");
+			}
+		}
+
+		task ("Change the order of the third page") {
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_itemName = "Test Page Name 2",
+				key_sourceItemName = "Test Page Name 3",
+				locator1 = "SiteNavigationMenu#ITEM_ENTRY_DRAG_ICON",
+				locator2 = "SiteNavigationMenu#ITEM_ENTRY");
+		}
+
+		task ("View the pages and children still are on the right indentation") {
+			for (var n : list "1,2,3") {
+				NavigationMenusAdmin.viewItemNotNested(itemName = "Test Page Name ${n}");
+			}
+
+			for (var n : list "1,2,3") {
+				NavigationMenusAdmin.viewItemNested(
+					groupName = "Test Site Name",
+					itemName = "Child Page ${n}",
+					parentItemName = "Test Page Name ${n}",
+					site = "true",
+					siteNavigationMenuName = "Navigation Menu Name");
+			}
+		}
+
+		task ("Put the third page on its original place") {
```

Comment:
Change back to the original order of the second parent page item and third parent page item